app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -450,6 +454,11 @@ private void onPreferenceClicked() {
         openPreferences();
     }
 
+    private void onExitClicked() {
+        finish();
+        moveTaskToBack(true);
```

Comment:
I don't think we need this after a finish?

Reply:
Okay, will refactor it later today 😄